backend/internal/adapters/secondary/repositories/gem.go

```diff
@@ -64,6 +67,48 @@ func (r *gemRepository) Add(gem *gems.Gem) (int64, error) {
 		return 0, fmt.Errorf("unable to add gem: %w", err)
 	}
 
+	var numGemsSentToGretel int64
+	sql, _, _ = qb.From(goqu.T("gems")).Select(goqu.COUNT("*")).Where(goqu.Ex{
+		"receiver_id": constants.GretelUserId,
+		"creator_id":  gem.CreatorId,
+	}).ToSQL()
+	_ = r.db.Get(&numGemsSentToGretel, sql)
+	isFirstGemSentToGretel := numGemsSentToGretel == 0
+
+	isNotCreatedByGretel := gem.CreatorId != constants.GretelUserId
+
+	isSentToGretel := gem.ReceiverId == constants.GretelUserId
+
+	shouldGretelSendGem := isNotCreatedByGretel && isSentToGretel && isFirstGemSentToGretel
+
+	// Recursive call; ensure that Gretel doesn't continuously send gems
+	// to herself
+	if shouldGretelSendGem {
+		// TODO: improve message phrasing and add attachment
```

Comment:
Is this meant to be addressed in this PR?

Reply:
hmm, not really haha. But i've added some placeholder images. Feel free to change them. Also, can I disable the init lint haha

Reply:
Okay!